assignments/array-methods.js

```diff
@@ -54,30 +54,56 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
+let fullName = []
+runners.forEach((name) => {
+  fullName.push(name.first_name + " " + name.last_name)
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+let allCaps = runners.map((name) => {
+  return name.first_name.toUpperCase()
+});
+
+console.log(allCaps);
 
 // ==== Challenge 3: Use .filter() ====
```

Comment:
good job. You've shown you're comfortable and knowledgable with higher order functions already so I'm glad to see you executing on these perfectly. Great job!